### ISSUE TITLE
A11y_.NET Core_ASP.NETCore_AngularCounter_NonTextContrast: The Color contrast ratio of the focus indicator of the "Increment" button is less than 3:1.

### ISSUE THREAD

**Issue by Truptik12** [dotnet/aspnetcore#42367]
![42367_A11y_ NET Core_ASP NETCore_AngularCounter_NonTextContrast](https://user-images.githubusercontent.com/86710491/175493511-c8129bc0-3d2f-43e7-ba90-5739297c414a.jpg)

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yWCAG2.1;#A11yTCS;#.NET Core[aspnetcore]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.4.11;#DesktopWeb;#Win11;#ColorContrast;#Benchmark;#A11ycaioh;#ceaccreviewed;

### Environment Details:
.NET Core
ASP.NET Core
Operating System: Windows 11 Enterprise 21H2
Chrome: Version 102.0.5005.115 (Official Build) (64-bit)

### Steps to Reproduce:

1. Open command prompt
2. Create a folder using command "mkdir foldername" e.g. "mkdir Angular"
3. Move to the created directory using command "cd foldername"
4. Run the following command "dotnet new angular -au Individual" and hit enter.
5. Run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. Browser page will open with above template loaded
9. Test the Counter Page
10. Turn on Color contrast analyzer and check that The Color contrast ratio of "Increment" is less than 3:1.

### Actual:
The Color contrast ratio of the focus indicator of the "Increment" button is less than 3:1.

**This issue is observed throughout the application for all the "Increment" buttons.**

### Expected:
The Color contrast ratio of the focus indicator of the  "Increment" button should be more than or equal to 3:1.

### User Impact:
Low vision users won't be able to see the focus indicator.

**Comment by SteveSandersonMS** [comment]
Fixed in https://github.com/dotnet/spa-templates/pull/70

**Comment by mkArtakMSFT** [comment]
#Closed;
